ivy/functional/backends/tensorflow/general.py

```diff
@@ -394,12 +394,15 @@ def scatter_nd(
         res = tf.tensor_scatter_nd_min(target, indices, updates)
     elif reduction == "max":
         res = tf.tensor_scatter_nd_max(target, indices, updates)
+    elif reduction == "mul":
+        updates = ivy.multiply(ivy.gather_nd(target, indices), updates).data
+        res = tf.tensor_scatter_nd_update(target, indices, updates)
```

Comment:
If this approach works correctly, I think you can do something similar for "mean" as well 🤔
Maybe even for the torch and paddle backends... 
If `scatter_nd` works correctly like this we can include all the modes for `put_along_axis` after all.